checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -885,3 +885,41 @@ actions:
         description: Success
         type: boolean
         required: true
+
+  check_if_address_in_group:
+    title: Check if Address in Group
+    description: Checks to see if an IP, CIDR, or domain is in an Address Group
+    inputs:
+      group:
+        title: Group
+        description: Group to check. UID is not supported
+        type: string
+        required: false
+        example: InsightConnect Block List
+        order: 1
+      address:
+        title: Address
+        description: Address to check in the group
```

Comment:
The Address Object name to check. If Enable Search is set to true then we search the addresses (IP, CIDR, doman) within the address object instead of matching address object name.